tests/roles/mariadb_copy/templates/post_checks.bash

```diff
@@ -5,17 +5,12 @@
 set +u
 . ~/.source_cloud_exported_variables_default
 set -u
-
 dbs=$(oc exec openstack-galera-0 -n $NAMESPACE -c galera -- mysql -rs -uroot -p"${PODIFIED_DB_ROOT_PASSWORD['super']}" -e 'SHOW databases;')
 echo $dbs | grep -Eq '\bkeystone\b' && echo "OK" || echo "CHECK FAILED"
-
-# ensure neutron db is renamed from ovs_neutron
 echo $dbs | grep -Eq '\bneutron\b'
-echo "${PULL_OPENSTACK_CONFIGURATION_DATABASES[@]}" | grep -Eq '\bovs_neutron\b' && echo "OK" || echo "CHECK FAILED"
-
-# ensure default cell is renamed to $DEFAULT_CELL_NAME, and the cell UUIDs retained intact
+echo "${PULL_OPENSTACK_CONFIGURATION_DATABASES[@]}" | grep -Eq '\bovs_neutron\b' && echo "OK" || echo "CHECK FAILED" # <1>
```

Comment:
note the magic, when copying code into text, you only need to bulk comment and replace to a ``$`` or ``>`` cursor for multi-line.
And uncomment ``# <x> ``footnotes to "activate" them as they are inactive and wouldn't work in shell.
 The result looks like commands, not scripts.
@jistr @klgill fyi